client/src/pages/ProfileList/List.tsx

```diff
@@ -0,0 +1,141 @@
+import CalendarTodayIcon from '@material-ui/icons/CalendarToday';
+import { CircularProgress } from '@material-ui/core';
+import { Rating } from '@material-ui/lab';
+import profilePhoto from '../../Images/d9fc84a0d1d545d77e78aaad39c20c11d3355074.png';
+import ProfilePhoto2 from '../../Images/775db5e79c5294846949f1f55059b53317f51e30.png';
+import moment from 'moment';
+import {
+  Grid,
+  Paper,
+  Typography,
+  Box,
+  Card,
+  TextField,
+  Button,
+  CardActions,
+  CardContent,
+  Divider,
+  CardMedia,
+  CardActionArea,
+  Avatar,
+  Input,
+} from '@material-ui/core';
+
+const placeholder = {
+  rating: 5,
+  price: 15,
+  location: 'Toronto, Ontario',
+};
+const defaultDescription = 'New pet sitter!';
+
+export default function List(): JSX.Element {
+  const classes = useStyles();
+  const [state, setState] = useState({
+    profiles: [],
+  });
+  const [selection, setSelection] = useState({
+    startDate: new Date(),
+    endDate: new Date(),
+    key: 'selection',
+  });
+  const profiles = state.profiles;
+  const [hidden, setHidden] = useState(true);
+
+  const handleSelect = (ranges: any) => {
+    setSelection(ranges.selection);
+  };
+  useEffect(() => {
+    function fetchProfile() {
+      axios.get('profile/all').then((response: any) => {
+        setState(response.data);
+      });
+    }
+    fetchProfile();
+  }, []);
+
+  if (!profiles) {
+    return <CircularProgress />;
+  } else {
+    return (
+      <Box>
+        <Grid justify="center" container sm>
+          <Grid item className={`${classes.searchContainer}`}>
+            <Typography align="center" variant="h4">
+              Your search results
+            </Typography>
+
+            <Paper className={classes.searchbar}>
+              <Grid alignItems="center" container>
+                <Grid item>
+                  <SearchIcon />
+                  <Input disableUnderline={true} placeholder="Search by location..."></Input>
+                </Grid>
+                <Grid item>
+                  <Box display="inline" className={classes.search}>
+                    <CalendarTodayIcon />
+                    <Button onClick={() => setHidden(!hidden)}>{`
+              ${moment(selection.startDate).format('DD-')}${`${moment(selection.endDate).format('DD MMMM YYYY')}`}
+                `}</Button>
+                  </Box>
+                </Grid>
+              </Grid>
+            </Paper>
+            <Box display="inline" className={classes.datePicker}>
+              <Paper hidden={hidden}>
+                <DatePicker handleSelect={handleSelect} />
+              </Paper>
+            </Box>
+          </Grid>
+        </Grid>
+        <Grid container justify="center" spacing={6} className={`${classes.root}`}>
+          {profiles.map((profile, key) => {
+            const { firstName, lastName, user, description } = profile;
+            return (
+              user && (
+                <Grid item key={key}>
```

Comment:
Using indexes as keys in React is considered an anti-pattern and not recommend. Please consider using a unique identifier as a key :)

Ref: https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9